test/playback.test.js

```diff
@@ -31,16 +37,23 @@ QUnit.module('Playback', {
     // videojs.log.level('debug');
     video.style = 'display: none;';
 
+    video.setAttribute('controls', '')
+    video.setAttribute('muted', '')
     video.width = 600;
     video.height = 300;
+    video.defaultPlaybackRate = 16;
+
     this.fixture.appendChild(video);
-    this.player = videojs(video, {
-      autoplay: 'muted'
-    });
-    this.player.ready(done);
+    this.player = videojs(video);
+
+    this.player.ready(done, true);
   },
   afterEach() {
     this.player.dispose();
+
+  },
+  after() {
+    document.body.removeChild(this.fixture);
```

Comment:
huh, apparently we don't have a global-shims file like we do in Video.js that sets up the qunit fixture https://github.com/videojs/video.js/blob/master/test/globals-shim.js

Reply:
There is a global `qunit-fixture` but its hidden by default. So if you had to go debug a specific unit test player and wanted to view it, it would be rather hard.

Reply:
Is that available in our version of qunit? It should be simple to show with css, if we wanted. However, I don't have a huge preference. Since this is how it's been going so far, probably fine to just leave it.

Reply:
I will see how easy it is to show, it is also in our version of qunit

Reply:
Ah, I guess that's a new thing? Guess we can remove it from global-shims in Video.js then.